cmd/thanos/query.go

```diff
@@ -52,6 +52,13 @@ import (
 	"github.com/thanos-io/thanos/pkg/ui"
 )
 
+const (
```

Comment:
nit: Maybe we should move these to a more common package since also the receiver uses these constants.